src/universal/components/SocketRoute/SocketRoute.js

```diff
@@ -42,6 +42,9 @@ const query = graphql`
               inviteeEmail
               teamId
               teamName
+              team {
```

Comment:
can you add this same request to `NotificationsAddedSubscription`? (I'll DRY it out later, but for now we've got 1 for the query, 1 for the subscription)